src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -7,4 +7,7 @@
         <!-- prevent unwanted caching when accessing via the web preview server -->
         <include path="/**" expiration="0s" />
     </static-files>
+      <manual-scaling>
+    <instances>1</instances>
```

Comment:
10/10 change

Reply:
It fixes all of our problems for now, but we'll need to figure out a way to enable scaling again if we want a lot of people to be able to use the site.